getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -121,12 +150,16 @@ Here's a blank C# script with some comments to demonstrate how it works.
         }
     }
 
-As you can see, the things normally in global scope in GDScript like Godot's ``print`` function are available in the ``GD`` namespace.
-For a list of those, see the class reference pages for :ref:`@GDScript <class_@gdscript>` and :ref:`@GlobalScope <class_@globalscope>`.
+As you can see, functions normally in global scope in GDScript like Godot's
+``print`` function are available in the ``GD`` namespace.
```

Comment:
`GD` is a class that is part of the `Godot` namespace.

It's not needed to enter in details here since the differences are already listed in its own page (to which I have to make some fixes), but I think this should also mention the `Mathf` class.